docs/usage/model_config.md

```diff
@@ -103,16 +103,33 @@ _(This script is complete, it should run "as is")_
 
 Here camel case refers to ["upper camel case"](https://en.wikipedia.org/wiki/Camel_case) aka pascal case
 e.g. `CamelCase`. If you'd like instead to use lower camel case e.g. `camelCase`,
-it should be trival to modify the `to_camel` funciton above.
+it should be trivial to modify the `to_camel` function above.
 
 ## Alias Precedence
 
-Aliases defined on the `Config` class of child models will take priority over any aliases defined on `Config` of a
-parent model:
+!!! warning
+    Alias priority logic changed in **v1.4** to resolve buggy and unexpected behaviour in previous versions.
+    In some circumstances this may represent a **breaking change**,
+    see [#1178](https://github.com/samuelcolvin/pydantic/issues/1177) and the precedence order below for details.
+
+A field alias's is chosen based on the following
```

Comment:
I think you meant `A field's alias` (or maybe this line is unnecessary/duplicated below)